source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -3647,6 +3675,10 @@ export class TextInput extends Input {
             result = document.createElement("input");
             result.className = this.hostConfig.makeCssClassName("ac-input", "ac-textInput");
             result.type = Enums.InputTextStyle[this.style].toLowerCase();
+
+            if (this.inputStyle !== null && this.inputStyle === Enums.InputStyle.ReadWrite && !this.inlineAction) {
```

Comment:
This code is repeated in multiple places. It should be implemented in the base `Input` class' `render' method and the style should be applied to whatever `internalRender` returns.